components/builder-api-client/src/artifactory.rs

```diff
@@ -32,8 +37,8 @@ use crate::{error::{Error,
             OriginKeyIdent,
             SchedulerResponse};
```

Comment:
We typically delete all the blank lines in between `use` clauses now and let `rustfmt` sort them.

Reply:
Will do